src/main/java/com/codeborne/selenide/Conditional.java

```diff
@@ -0,0 +1,12 @@
+package com.codeborne.selenide;
+
+import java.time.Duration;
+import java.util.function.Predicate;
+
+public interface Conditional<T> {
+
+  void should(Predicate<T> predicate, String message);
+
+  void should(Predicate<T> predicate, String message, Duration timeout);
```

Comment:
Look like you're missing condition with `polling`

Reply:
I thought that polling is not so frequent configurable but if you say, i'll do it

Reply:
There is no sense to make ability to configure pooling and not extend  interface with it